lib/dialects/mysql/query/compiler.js

```diff
@@ -18,6 +18,47 @@ class QueryCompiler_MySQL extends QueryCompiler {
     this._emptyInsertValue = '() values ()';
   }
 
+  // Compiles an `insert` query, allowing for multiple
+  // inserts using a single query statement.
+  insert() {
+    let sql = QueryCompiler.prototype.insert.call(this);
```

Comment:
here we can use super.insert() for simplicity

Reply:
Ah yeah, I just copy-pasted that from somewhere else. Have changed it to use super.

Reply:
Can you also fix other places?